test/cholmod.jl

```diff
@@ -238,11 +238,12 @@ end
     @test_throws CHOLMOD.CHOLMODException CHOLMOD.Sparse(p)
 end
 
+# Pass 8 instead of CHOLMOD_INTLONG which no longer exists.
 @testset "illegal itype I" begin
     p = Ti == Int64 ? cholmod_l_allocate_sparse(1, 1, 1, true, true, 0, CHOLMOD_REAL, getcommon(Ti)) :
         cholmod_allocate_sparse(1, 1, 1, true, true, 0, CHOLMOD_REAL, getcommon(Ti))
     puint = convert(Ptr{UInt32}, p)
-    unsafe_store!(puint, CHOLMOD_INTLONG, 3*div(sizeof(Csize_t), 4) + 5*div(sizeof(Ptr{Cvoid}), 4) + 2)
+    unsafe_store!(puint, 123, 3*div(sizeof(Csize_t), 4) + 5*div(sizeof(Ptr{Cvoid}), 4) + 2)
```

Comment:
The above comment suggests that this should be "8" (not "123"). (But admittedly, I don't know what any of this means...)

Reply:
@rayegun you changed this in #480

Reply:
I believe this test is checking that we get the right exception thrown from the wrapper when the int value is incorrect. In old versions, `INT_LONG` would throw that exception, but now that was removed, this looks to just be a random number that will throw the exception still.

Reply:
Should we just remove this testset then?

Reply:
If there is no other test that checks that the correct exception is thrown, it might be worth keeping this.
But maybe, use the same (bogus) integer in the comment above this test to avoid confusion in the future. It might also help to point out that the integer is one that isn't valid input.

Reply:
> But maybe, use the same (bogus) integer in the comment above this test to avoid confusion in the future.

Which comment are you referring to?

Reply:
The one in the new line 241:
```
# Pass 8 instead of CHOLMOD_INTLONG which no longer exists.
```

Reply:
Sorry this was definitely a "get it work" change. There should just be a comment above saying that we're using a bogus Itype to ensure an exception is correctly thrown. We still want to test it, I'm not sure if it should be `8` or some random integer like `123`.

Reply:
I improved the comments (I hope).  How does it look now?

Reply:
Looks good to me. Thanks! 👍

Reply:
Thanks for the help!